project-management/src/main/java/life/qbic/projectmanagement/application/measurement/Labeling.java

```diff
@@ -7,6 +7,6 @@
  *
  * @since 1.0.0
  */
-public record Labeling(String sampleCode, String labelType, String label) {
+public record Labeling(String labelType, String label) {
```

Comment:
Please adapt the javadoc to reflect the change. This record no longer associates sample code and labelling but only represents labelling.